docs/developer-docs/latest/development/custom-fields.md

```diff
@@ -176,15 +176,40 @@ The `Input` React component receives several props. The [`ColorPickerInput` file
 
 ### Options
 
-`app.customFields.register()` can pass an additional `options` object with the following parameters:
+Custom fields can have their own settings in the Content-type builder. In
+`app.customFields.register()` we can pass an additional `options` object  with the following parameters enable their use.
 
 | Options parameter | Description                                                                     | Type                    |
 | -------------- | ------------------------------------------------------------------------------- | ----------------------- |
 | `base`         | Settings available in the _Base settings_ tab of the field in the Content-type Builder       | `Object` or  `Array of Objects` |
 | `advanced`     | Settings available in the _Advanced settings_ tab of the field in the Content-type Builder   | `Object` or  `Array of Objects` |
 | `validator`    | Validator function returning an object, used to sanitize input. Uses a [`yup` schema object](https://github.com/jquense/yup/tree/pre-v1).  | `Function`              |
 
-Both `base` and `advanced` settings accept an object or an array of objects, each object being a settings section. Each settings section could include:
+
+By default, Strapi has options available for it's [field model](/developer-docs/latest/backend-customization/models.md#validations) you can use them in your custom field by copying the object value from [this file](https://github.com/strapi/strapi/blob/e033275bd0915049fa4fa823f00c17ad53a8fc44/packages/core/content-type-builder/admin/src/components/FormModal/attributes/attributeOptions.js#L1) and respecting the structure below. 
+
+```javascript
+{
+	// The option name
+  name: 'required', 
+	// The input type for this option
+  type: 'checkbox',
+	// The default value for this option
+	defaultValue: true,
+	// The translation object for this option's label
+  intlLabel: {
+    id: getTrad('form.attribute.settings.default'),
+    defaultMessage: 'Default value',
+  },
+	// The translation object for this option's description
+	intlDescription: {
+    id: getTrad('form.attribute.settings.default.description'),
+    defaultMessage: 'Default value description',
+	}
+}
+```
+
+Should you not want to use the defaults, `options` enables the creation of Custom settings when we set up or edit a custom field in the Content-types Builder. Both `base` and `advanced` settings accept an object or an array of objects, each object being a settings section. Each settings section could include:
```

Comment:
```suggestion
If defaults are not appropriate for your custom field use case, `options` enables the creation of custom settings when setting up or editing a custom field in the Content-types Builder. Both `base` and `advanced` settings accept an object or an array of objects, each object being a settings section. Each settings section could include:
```

Reply:
Also do y'all think we should mention here that base and advanced refer to the basic and advanced tabs in the CTB modal form?

Reply:
> Both `base` and `advanced` settings accept an object or an array of objects

I believe they only accept an array of objects 🤔